docs/guides/runtime-options-overview.mdx

```diff
@@ -18,288 +19,308 @@ service = QiskitRuntimeService(channel="ibm_cloud", channel_strategy="q-ctrl")
 
-```python
-from qiskit_ibm_runtime import Options
+``` python
+from qiskit_ibm_runtime.options import EstimatorOptions
 
-options = Options()
-options.resilience_level = 1
-options.execution.shots = 2048
-```
-## Pass options to a primitive
+# Set options during primitive initialization with a nested dictionary
+estimator1 = Estimator(backend, options={"twirling": {"enable_gates": True}})
 
-### Options class
+# Update the options after initialization
+estimator1.options.twirling.num_randomizations = 100
 
-When creating an instance of the `Estimator` or `Sampler` class, you can pass in the `options` you created in the options class. Those options will then be applied when you use `run()` to perform the calculation.  Example:
+# Set options during primitive initialization with an options class
+options = EstimatorOptions()
+options.twirling.enable_gates = True
+estimator2 = Estimator(backend, options=options)
 
-<Tabs>
-  <TabItem value="PrimV2" label="V2 primitives">
-    `SamplerV2` and `EstimatorV2` have separate options classes that do not need to be instantiated.  You can see the available options and update option values during or after primitive initialization.  Those options will then be applied when you use `run()` to perform the calculation.  Example:
+# Update the options after initialization
+estimator2.options.twirling.num_randomizations = 100
 
-```python
-estimator = Estimator(mode=backend)
+# Keep all default options during primitive initialization
+estimator3 = Estimator(backend)
 
-# Setting options after primitive initialization
-# This uses auto complete.
-estimator.options.default_shots = 4000
+# Update the options after initialization
+estimator3.options.twirling.enable_gates = True
+estimator3.options.twirling.num_randomizations = 100
 ```
-  </TabItem>
 
-  <TabItem value="PrimV1" label="V1 primitives">
-When creating an instance of the `Estimator` or `Sampler` class, you can pass in the `options` you created in the options class. Those options will then be applied when you use `run()` to perform the calculation.  Example:
+### Special cases
 
-```python
-estimator = Estimator(backend=backend, options=options)
-result = estimator.run(circuit, observable).result()
-print(f">>> Metadata: {result.metadata[0]}")
-```
-  </TabItem>
-</Tabs>
+#### Resilience level (Estimator only)
 
-### Primitive initialization
+The resilience level is not actually an option, but is a specification of a base set of curated options to build off of. In general, level 0 turns off all error mitigation, level 1 turns on options for measurement error mitigation, and level 2 turns on options for gate and measurement error mitigation.
```

Comment:
I get where this is coming from, but I think it'd be confusing to say it's not an option when it's inside `EstimatorOption`

```suggestion
The resilience level is not an option that directly impacts the primitive query. It is instead a specification of a base set of curated options to build off of. In general, level 0 turns off all error mitigation, level 1 turns on options for measurement error mitigation, and level 2 turns on options for gate and measurement error mitigation.
```

Reply:
This is one of the reasons I don't like resilience levels. It really, really isn't like the other options, which is why I don't like to think of it as an option.